client/account-recovery/account-recovery-root/index.jsx

```diff
@@ -15,33 +16,75 @@ import DocumentHead from 'components/data/document-head';
 import LostPasswordForm from 'account-recovery/lost-password-form';
 import ForgotUsernameForm from 'account-recovery/forgot-username-form';
 import ResetPasswordForm from 'account-recovery/reset-password-form';
+import ResetPasswordEmailForm from 'account-recovery/reset-password-email-form';
+import ResetPasswordSmsForm from 'account-recovery/reset-password-sms-form';
+import ResetPasswordConfirmForm from 'account-recovery/reset-password-confirm-form';
 import { ACCOUNT_RECOVERY_STEPS as STEPS } from 'account-recovery/constants';
 import {
 	isAccountRecoveryResetOptionsReady,
 	isAccountRecoveryUserDataReady,
+	getAccountRecoveryResetSelectedMethod,
+	getAccountRecoveryValidationKey,
 } from 'state/selectors';
 
 const getPageInfo = ( translate, step ) => {
+	const concatHeadTitle = ( parentTitle, childTitle ) => ( parentTitle + ' ‹ ' + childTitle );
+
 	const pageInfo = {
 		[ STEPS.LOST_PASSWORD ]: {
 			trackerTitle: 'Account Recovery > Lost Password',
-			documentHeadTitle: translate( 'Lost Password ‹ Account Recovery' ),
+			documentHeadTitle: concatHeadTitle( translate( 'Lost Password' ), translate( 'Account Recovery' ) ),
 		},
 		[ STEPS.FORGOT_USERNAME ]: {
 			trackerTitle: 'Account Recovery > Forgot Username',
-			documentHeadTitle: translate( 'Forgot Username ‹ Account Recovery' ),
+			documentHeadTitle: concatHeadTitle( translate( 'Forgot Username' ), translate( 'Account Recovery' ) ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Forgot your username?' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).